src/main/java/sirius/biz/storage/layer3/DeleteFilesJob.java

```diff
@@ -169,7 +179,14 @@ private boolean handleFile(VirtualFile file) {
         }
 
         if (file.readOnly()) {
-            process.log(ProcessLog.warn().withNLSKey("DeleteFileJob.file.readOnly").withContext(PATH_KEY, file.path()));
+            process.log(ProcessLog.warn()
+                                  .withNLSKey("DeleteFilesJob.file.readOnly")
+                                  .withContext(PATH_KEY, file.path()));
+            return false;
+        }
+
+        if (onlyUnused && vfs.isInUse(file)) {
+            process.log(ProcessLog.warn().withNLSKey("DeleteFilesJob.file.inUse").withContext(PATH_KEY, file.path()));
```

Comment:
Wont this possibly log a lot of messages?

Reply:
Especially in OXOMI, it could. I think is important to list them. Should we consider logging to a CSV instead?

Reply:
Or we keep it but reduce the persistence period to 1 or 3 months, so the messages are deleted sooner.